src/modules/settings/pages/about.jsx

```diff
@@ -0,0 +1,186 @@
+import React, {useRef, useState} from 'react';
+import {save} from 'save-file';
+
+import IconButton from 'rsuite/lib/IconButton/index.js';
+import PanelGroup from 'rsuite/lib/PanelGroup/index.js';
+import Panel from 'rsuite/lib/Panel/index.js';
+import Icon from 'rsuite/lib/Icon/index.js';
+import {faUpload} from '@fortawesome/free-solid-svg-icons/faUpload';
+import {faRedo} from '@fortawesome/free-solid-svg-icons/faRedo';
+import {faDownload} from '@fortawesome/free-solid-svg-icons/faDownload';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import settings from '../../../settings.js';
+import storage from '../../../storage.js';
+import debug from '../../../utils/debug.js';
+import header from '../styles/header.module.css';
+import styles from '../styles/about.module.css';
+import {DefaultValues, SettingIds} from '../../../constants.js';
+import CloseButton from '../components/CloseButton.jsx';
+
+function isJSON(string) {
+  try {
+    JSON.parse(string);
```

Comment:
why do we parse the json twice.. perhaps we can return the parsed data or return null?

Reply:
This was existing code, but I agree.